syntheticsclientv2/common_models.go

```diff
@@ -77,6 +77,8 @@ type StepsV2 struct {
 	URL                string  `json:"url,omitempty"`
 	Action             string  `json:"action,omitempty"`
 	WaitForNav         bool    `json:"waitForNav"`
+  WaitForNavTimeout  int     `json:"waitForNavTimeout"`
```

Comment:
Thank you for adding this!
Mind hitting these with a linter or just fixing the spacing? 

Sort of surprised our pipeline didn't catch this. May need to tweak that step of the workflow to fail when linting isn't correct.

Reply:
Hah ok so when I pull the PR as a branch this all looks fine. And Sang see's the same. Lint doesn't show any changes. This must be a Github weirdness?

Ok to merge imo and if it ends up being weird...  can fix in a later PR IMO.

Reply:
(discussed over slack)

i tried `golangci-lint run`, but it didn't change any files. 
![image](https://github.com/user-attachments/assets/15418793-fb98-49ac-8eed-40ef3d568a21)